src/platform/packages/shared/kbn-esql-validation-autocomplete/src/validation/validation.ts

```diff
@@ -225,6 +226,16 @@ function validateCommand(
       messages.push(...joinCommandErrors);
       break;
     }
+    case 'fork': {
```

Comment:
Maybe it is already covered by the parser, if not, we could check here if only the allowed sub-query commands are used.

Reply:
It's covered in the grammar